### ISSUE TITLE
Age support for age stratification

### ISSUE THREAD

**Issue by ryantibs** [cmu-delphi/www-covidcast#1138]
Our newest Quidel test positivity signals will be broken down by age groups: 0-4, 5-17, 0-17, 18-49, 50-64, 65+.  That is, these will be provided in addition to the original all-ages signal.

I would like the time series plots for Quidel test positivity on the COVIDcast dashboard to allow for viewing the signal broken down by age group.  I'm thinking something simple like this: 

<img width="1180" alt="COVIDcast  COVID Antigen Test Positivity Indicator Details" src="https://user-images.githubusercontent.com/732541/149988835-b9ddd8c0-eb39-4e8d-b409-9c7034705377.png">

- We add a side toggle called "Age Groups", which defaults to being "off", so the default view is as it is now.  
- There are checkboxes below it that are all disabled (grayed out) and unchecked when the "Age Groups" toggle is off.  
- There is one checkbox for the "All Ages" group and then one checkbox per proper age group.  Each one also shows the color of the corresponding line.  (Colors here are just chosen as a demo; I'm not suggesting we actually use them.)
- When "Age Groups" is toggled "on", then you all these checkboxes are enabled and checked by default.  So you get the "All Ages" signal and every proper age group signal all overlaid, on the time series plot.
- When "Age Groups" is toggled "on", then the "Raw Data" toggle beneath the plot is disabled (no way to display raw data and multiple signals at once, would be too crowded), but the "Show All Dates" toggle is still enabled.  
- Note: while we're at it, I'd like to rename "Show All Dates" to be "All Dates", throughout the dashboard.  That would be consistent with "Raw Data" and "Age Groups" (where we don't start with a verb).  Also, to be clear, the "Include 0 in Y-Axis" toggle will be going away with v3.1, which hasn't been pushed live yet, and I got this screenshot from the live dashboard.

@sgratzl Do you need any other points of clarification and can you give it a shot?

**Comment by ryantibs** [comment]
Each age-group would be its own signal under the Quidel data sources, see https://github.com/cmu-delphi/covidcast-indicators/pull/1467

**Comment by sgratzl** [comment]
AFAIK there are not part of the /covidcast/meta meta data yet

**Comment by krivard** [comment]
correct, we don't have advanced meta for the new Quidel signals yet. They are still unannounced while we resolve what went wrong with the "mark as deleted" deployment. I have however directed Jingjing to fill out the new advanced meta so it'll be ready when we are. [preview here](https://docs.google.com/spreadsheets/d/1zb7ItJzY5oq1n-2xtvnPBiJu2L3AqmCKubrLkKJZVHs/edit#gid=1955161993)

**Comment by krivard** [comment]
[Unblocked](https://api.covidcast.cmu.edu/epidata/covidcast/meta?signal=quidel-covid-ag:covid_ag_smoothed_pct_positive_age_0_4)
```
[{
    "db_source": "quidel",
    "description": "Quidel, Inc. manufactures diagnostic equipment for healthcare applications, and provides Delphi with anonymized data on tests and test results. This source includes antigen tests for COVID-19.",
    "dua": "https://cmu.box.com/s/sax48yxnahllrnbqlq6wqxblg6lsyq24",
    "license": "CC BY",
    "link": [{
        "alt": "API Documentation",
        "href": "https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/quidel.html#covid-19-tests"
    }],
    "name": "Quidel Inc. (COVID)",
    "reference_signal": "covid_ag_smoothed_pct_positive",
    "signals": [
      {
        "active": true,
        "category": "cases_testing",
        "compute_from_base": false,
        "description": "When a patient (whether at a doctorâs office, clinic, or hospital) has COVID-like symptoms, doctors may order an antigen test. An antigen test can detect parts of the virus that are present during an active infection. This is in contrast with antibody tests, which detect parts of the immune system that react to the virus, but which persist long after the infection has passed. For this signal, we compute the percentage of antigen tests performed that were positive for COVID-19 among people ages 0-4, smoothed using a 7-day moving average and geographical pooling",
        "format": "percent",
        "geo_types": {
          "county": {"max":62.14957746600657, "mean":10.0007593, "min":0.1237623762376237, "stdev":7.8936033},
          "hhs": {"max":32.2742475, "mean":8.2142316, "min":0.1201923076923077, "stdev":5.2990461},
          "hrr": {"max":62.51885369532428, "mean":9.8405488, "min":0.1237623762376237, "stdev":7.5701974},
          "msa": {"max":59.1437924, "mean":9.8623176, "min":0.1237623762376237, "stdev":7.7046062},
          "nation": {"max":28.374671, "mean":8.8271718, "min":2.5985771132280235, "stdev":4.8385832},
          "state": {"max":42.4667651, "mean":8.7505136, "min":0.1269035532994923, "stdev":6.2076694}
        },
        "has_sample_size": true,
        "has_stderr": true,
        "high_values_are": "bad",
        "is_cumulative": false,
        "is_smoothed": true,
        "is_weighted": false,
        "link": [{
            "alt": "Technical description",
            "href": "https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/quidel.html#covid-19-tests"
        }],
        "max_issue": 20220227,
        "max_time": 20220222,
        "min_time": 20200526,
        "name": "COVID-19 Antigen Tests: Percent Positive, Ages 0-4 (Smoothed)",
        "short_description": "Percentage of antigen tests that were positive for COVID-19 among people ages 0-4",
        "signal": "covid_ag_smoothed_pct_positive_age_0_4",
        "signal_basename": "covid_ag_raw_pct_positive_age_0_4",
        "source": "quidel-covid-ag",
        "time_label": "Date",
        "time_type": "day",
        "value_label": "Percentage"
      }
    ],
    "source": "quidel-covid-ag"
}]
```
